keyboards/column13/modulo/expander.c

```diff
@@ -0,0 +1,69 @@
+//
```

Comment:
This appears to be a re-implementation of https://github.com/qmk/qmk_firmware/blob/master/drivers/gpio/pca9555.c

If so, please use that instead.